WD-Firmware/psp/api_inc/psp_api.h

```diff
@@ -28,9 +28,12 @@
 */
 
 #include "psp_config.h"
-#include "psp_macro.h"
 #include "psp_defines.h"
 #include "psp_interrupt_api.h"
+#include "psp_macros.h"
+#include "psp_attributes.h"
+#include "psp_intrinsics.h"
```

Comment:
move #include "psp_attributes.h" to be the last include

Reply:
done

Reply:
https://github.com/westerndigitalcorporation/riscv-fw-infrastructure/pull/12/commits/6c26f8aba68f07621c53b564702cbab8a7d91dd1